tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -1043,6 +1043,7 @@ def fit(
 
         verbosity = kwargs.get("verbosity", self.verbosity)
         set_logger_verbosity(verbosity)
+        warn_if_mlflow_autologging_is_enabled(logger=logger)
```

Comment:
is it enough to call this in `fit`? why not somewhere in an __init__ file?

Reply:
The patched metrics only cause problems when saving objects with pickle (which is only required during training). Calling methods like `leaderboard` or `evaluate` should work normally.